packages/dashboard/src/components/palette/icons/input-component.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const InputComponent: React.FC = () => <div>input</div>;
```

Comment:
Something to note / not required, you could copy and paste the text icon and just change the T to an I

Reply:
I like that idea! Let me run it by UX. Thanks.